packages/gatsby-theme-docs/src/modules/ai-assistant/components/chat.utils.js

```diff
@@ -20,10 +20,6 @@ const isAllowedEmailDomain = (email, domainList) => {
   return allowedEmailDomainRegExp.test(email);
 };
 
-export const isAllowedUser = (user) => {
```

Comment:
Not sure if this has been discussed. But `isAllowedUser` had been added to give the possibility to whitelist only specific email domains (customers). I guess this is not necessary anymore as the go live will be for all customers. But worth to check @nkuehn

Reply:
Yes, but not yet.  We'll do that only shortly before go-live. 

But we'll likely need a denylist on the API server to handle known misuse attempts.

Reply:
I'll add a ticket in the go-live, for now, let's leave it @timonrey

Reply:
I removed the `isAllowedUser` because it was not used anywhere anymore and there is also an identical negated value for isNotAllowedUser already.

Reply:
I think you might got confused with `isNotValidatedUser` which has a different meaning.
`isAllowedUser` IMHO should stay preventing the chat to be launched if the user is not in the whitelist, up to you where you want to use it (either in the launch button or in the modal), it's temporary anyway we'll remove it when we go finally live

Reply:
Maybe it has a different meaning, but the function is identical to `isAllowedUser`, just returning false of user is allowed. So I used it instead of `isAllowedUser`.
https://github.com/commercetools/commercetools-docs-kit/pull/1892/files#diff-e625f24fbc71d5d652eca48d7d23c61d1d019faf2a0ddb4667bd82395c7954f0L27